modules/shared/partials/acid-transactions.adoc

```diff
@@ -146,8 +158,6 @@ If two such writes *do* conflict then the transactional write will 'win', overwr
 
 Note this only applies to _writes_.
 Any non-transactional _reads_ concurrent with transactions are fine, and are at a Read Committed level.
-
-To help detect that this co-operative requirement is fulfilled, the application can subscribe to the client's event logger and check for any `IllegalDocumentState` events, like so:
```

Comment:
To check, this line will be moved back into the Java docs?

Reply:
Yep! That's the intention.